pbf/templates/player.html

```diff
@@ -205,13 +205,27 @@ <h4>Impending:</h4>
   <ul>
     <div class="container-fluid">
       <div class="row">
-	{% for card in player.impending.all %}
+	{% for i in impending_with_energy.all %}
```

Comment:
```suggestion
	{% for i in player.gameplayerimpendingwithenergy_set.all %}
```

Reply:
Cool, yea, I had tried your original suggestion and had to step away as it wasn't working. I'll try to clean up the naming and retest everything when I have a moment later.